build.sbt

```diff
@@ -61,6 +59,22 @@ lazy val testScalastyle = taskKey[Unit]("testScalastyle")
 testScalastyle := scalastyle.in(Test).toTask("").value
 (test in Test) := ((test in Test) dependsOn testScalastyle).value
 
+/***************************
+* Spark Packages settings *
+***************************/
+ 
+name := "hyperspace-core"
+
+spName := "hyperspace-core"
+
+spAppendScalaVersion := true
+
+spIncludeMaven := true
+
+spIgnoreProvided := true
+
+packageBin in Compile := spPackage.value
+
```

Comment:
For delta, I see
```
spName := "databricks/delta-core"

sparkComponents := Seq("sql")
```
Can you explain the difference?

Reply:
Looking at https://github.com/databricks/sbt-spark-package#spark-package-developers
I think we can do "microsoft/hyperspace-core"?
Also, is `sparkComponents` a dependency for this project, meaning we can remove spark from `libraryDependencies`?

Reply:
hmm. In delta sbt script, They still have library dependencies https://github.com/delta-io/delta/blob/master/build.sbt#L26
Document for the plug in says this is only for spark packages. 
In my understanding, since we have also test package dependencies which I sparkComponents may not be able to cover.
  "org.apache.spark" %% "spark-core" % sparkVersion % "test" classifier "tests",
 

I can try to rename to microsoft/hyperspace-core following same convention.